website/docs/modules/create.html.markdown

```diff
@@ -118,8 +118,12 @@ standard structure.
 
 * **README**. The root module and any nested modules should have README
   files. This file should be named `README` or `README.md`. The latter will
-  be treated as markdown. The README doesn't need to document inputs or
-  outputs of the module because tooling will automatically generate this.
+  be treated as markdown. There should be a description of the module and 
+  what it should be used for, as well as a basic usage example.  If possible,
+  a visual diagram of the modules should be included. The README doesn't need 
```

Comment:
"Consider including a visual diagram depicting the infrastructure resources the module may create and their relationship."

Maybe?